user-story/src/App.tsx

```diff
@@ -0,0 +1,21 @@
+import { BrowserRouter, Route, Routes } from 'react-router-dom'
+import './App.css'
+import Home from './pages/Home'
+import Dashboard from './pages/Dashboard'
+import Recent from './pages/Recent'
+import Info from './pages/Info'
+import Setting from './pages/Setting'
+
+export const App = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/dashboard" element={<Dashboard />} />
+        <Route path="/info" element={<Info />} />
+        <Route path="/recent" element={<Recent />} />
+        <Route path="/setting" element={<Setting />} />
```

Comment:
create a separate variable for routes and get the path value

Reply:
Address